### PR TITLE
Update the Storybook React in PT to match the most recently in EN

### PR DESCRIPTION
**Pull request by elfiservice**
I did some adjusts following the EN (English) that is in production
- [https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/](https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/)

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@elfiservice thank you for the time and effort you put into the pull request. But instead of just one chapter, we'd love to get the Portuguese translations updated to the latest version of the content. If you're willing to work on it, could you please adjust the title of the pull request, and continue working on it and let me know once you're done.


One small comment to help you get this merged faster. If you don't mind leave the code's comments in their natural form (English). I know that translating them, brings it closer to home. But just for consistency purposes leave it as is.

Let me know.

Stay safe

File: content/intro-to-storybook/react/pt/get-started.md [hunk 1]
```diff
@@ -6,48 +6,48 @@ commit: 'ac1ec13'
...
-# Run the frontend app proper on port 3000:
+# Execute o próprio aplicativo frontend na porta 3000:
 yarn start
 ```
 
-<div class="aside">
-Pode ter reparado que foi adicionada a opção <code>--watchAll</code> no comando de testes, não se preocupe, é intencional. Esta pequena alteração irá garantir que todos os testes sejam executados e que a nossa aplicação está a funcionar corretamente. À medida que progride no tutorial, irão ser apresentados diversos cenários de testes, talvez queira considerar e adicionar esta opção ao script de testes no ficheiro (ou arquivo) <code>package.json</code> para garantir que todos os testes sejam executados.
+<div class="aside"> 
+💡 Veja o sinalizador (flag) <code>--watchAll</code> no comando de teste, incluindo este sinalizador garante que todos os testes sejam executados. Enquanto avança neste tutorial, você será apresentado a diferentes cenários de testes. Você pode considerar querer ajustar seus scripts <code>package.json</code> de acordo.
 </div>
```
Comment (K-Schaeffer) [inline content/intro-to-storybook/react/pt/get-started.md:51]:
```suggestion
💡 Note que a flag <code>--watchAll</code> no comando de teste garante que todos os testes sejam executados. Enquanto avança neste tutorial, você será apresentado a diferentes cenários de testes. Você pode considerar querer ajustar seus scripts no <code>package.json</code> de acordo.
</div>
```

**Review by K-Schaeffer — COMMENTED** [review]
Some suggestion that improve the legiability in portuguese.

**Comment by elfiservice** [comment]
tks @K-Schaeffer  for the review 🙏🏽

**Comment by jonniebigodes** [comment]
@elfiservice if you don't mind updating the Get started section based on #504 we'd appreciate it. I'm going to take a look at it sometime next week and if anything stands out, I'll gladly merge it and release the updated version.

Let me know once you've taken care of this and we'll go from there.

**Comment by elfiservice** [comment]
> @elfiservice if you don't mind updating the Get started section based on #504 we'd appreciate it. I'm going to take a look at it sometime next week and if anything stands out, I'll gladly merge it and release the updated version.
> 
> Let me know once you've taken care of this and we'll go from there.
Hi @jonniebigodes , lets update :)
- What I need to do?

**Comment by jonniebigodes** [comment]
@elfiservice what I meant with my previous comment was to include the instructions to set the branch to main in the Get started section. More specifically in "Confirmar as alterações" heading. As a reference see how it's mentioned in the [English version](https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/)

**Comment by jonniebigodes** [comment]
@elfiservice Closing this for now as it seems that there's been no movement from your end for quite some time. If you are able to update the Portuguese translation, please feel free to open a new pull request and ping me for a review and we'll take it from there.

Looking forward to seeing this translation get the much-deserved updates it requires.

Hope you have a great week.

Stay safe

**Pull request closed without merging**